composer.json

```diff
@@ -29,7 +29,10 @@
     "symfony/process": "~2.1||~4.1.0",
     "symfony/yaml": "^3.3||^4.0",
     "twig/twig": "^1.0||^2.0",
-    "magento/magento-cloud-components": "^1.0"
+    "magento/magento-cloud-components": "^1.0",
+
+
+    "magento/framework": "*"
```

Comment:
This will be removed after `magento-cloud-components` release

Reply:
Remove excess lines

Reply:
Just wanted to separate somehow packages from the package that will be removed. Lines will be removed too.

Reply:
It's better to create a task